modules/administration-guide/partials/proc_collecting-che-metrics-with-prometheus.adoc

```diff
@@ -7,7 +7,7 @@ This section describes how to use the Prometheus monitoring system to collect, s
 
 .Prerequisites
 
-* {prod-short} is exposing metrics on port `8087`. See xref:enabling-and-exposing-{prod-id-short}-metrics_{context}[Enabling and exposing che metrics].
+* {prod-short} is exposing metrics on port `8087`. See xref:enabling-and-exposing-{prod-id-short}-metrics_{context}[Enabling and exposing {prod-short} metrics].
```

Comment:
{prod-short} is not rendered correctly here?

Reply:
@ibuziuk What do you mean? The change displays correctly on a local build.